src/main/app/index.ts

```diff
@@ -5,6 +5,11 @@
 
 import {initialize} from './initialize';
 
+// TODO: Singletons, we need DI :D
```

Comment:
This is a temporary solution. The best way to solve this would be a DI framework, but that might be a bit overkill for our application.

A quicker solution may be to just have a file that imports all the singletons when the app starts instead of having them all here.

Reply:
we could consider a DI framework if this grows like crazy, so far with just three I don't think is needed either